File: app/src/main/java/sagsaguz/brillavedicmaths/utils/BrillaVMUsersDO.java
package sagsaguz.brillavedicmaths.utils;

import com.amazonaws.mobileconnectors.dynamodbv2.dynamodbmapper.DynamoDBAttribute;
import com.amazonaws.mobileconnectors.dynamodbv2.dynamodbmapper.DynamoDBHashKey;
import com.amazonaws.mobileconnectors.dynamodbv2.dynamodbmapper.DynamoDBIndexHashKey;
import com.amazonaws.mobileconnectors.dynamodbv2.dynamodbmapper.DynamoDBIndexRangeKey;
import com.amazonaws.mobileconnectors.dynamodbv2.dynamodbmapper.DynamoDBRangeKey;
import com.amazonaws.mobileconnectors.dynamodbv2.dynamodbmapper.DynamoDBTable;

import java.util.List;
import java.util.Map;
import java.util.Set;

@DynamoDBTable(tableName = "brillavm-mobilehub-86056955-BrillaVMUsers")

public class BrillaVMUsersDO {
    private String _phone;
    private String _email;
    private String _age;
    private String _city;
    private String _name;

    @DynamoDBHashKey(attributeName = "phone")
    @DynamoDBAttribute(attributeName = "phone")
    public String getPhone() {
        return _phone;
    }

    public void phone(final String _phone) {
        this._phone = _phone;
    }
    @DynamoDBRangeKey(attributeName = "email")
    @DynamoDBAttribute(attributeName = "email")
    public String getEmail() {
        return _email;
    }

    public void email(final String _email) {
        this._email = _email;
    }
    @DynamoDBAttribute(attributeName = "age")
    public String getAge() {
        return _age;
    }

    public void age(final String _age) {
        this._age = _age;
    }
    @DynamoDBAttribute(attributeName = "city")
    public String getCity() {
        return _city;
    }

    public void city(final String _city) {
        this._city = _city;
    }
    @DynamoDBAttribute(attributeName = "name")
    public String getName() {
        return _name;
    }

    public void name(final String _name) {
        this._name = _name;
    }

}

File: app/src/main/java/sagsaguz/brillavedicmaths/utils/BrillaVMUsersDO.kt
package sagsaguz.brillavedicmaths.utils

import com.amazonaws.mobileconnectors.dynamodbv2.dynamodbmapper.DynamoDBAttribute
import com.amazonaws.mobileconnectors.dynamodbv2.dynamodbmapper.DynamoDBHashKey
import com.amazonaws.mobileconnectors.dynamodbv2.dynamodbmapper.DynamoDBIndexHashKey
import com.amazonaws.mobileconnectors.dynamodbv2.dynamodbmapper.DynamoDBIndexRangeKey
import com.amazonaws.mobileconnectors.dynamodbv2.dynamodbmapper.DynamoDBRangeKey
import com.amazonaws.mobileconnectors.dynamodbv2.dynamodbmapper.DynamoDBTable

@DynamoDBTable(tableName = "your table name as in dynamo db")
class BrillaVMUsersDO {
    @get:DynamoDBHashKey(attributeName = "phone")
    @get:DynamoDBAttribute(attributeName = "phone")
    var phone: String? = null
        private set
    @get:DynamoDBRangeKey(attributeName = "email")
    @get:DynamoDBAttribute(attributeName = "email")
    var email: String? = null
        private set
    @get:DynamoDBAttribute(attributeName = "age")
    var age: String? = null
        private set
    @get:DynamoDBAttribute(attributeName = "city")
    var city: String? = null
        private set
    @get:DynamoDBAttribute(attributeName = "name")
    var name: String? = null
        private set
    @get:DynamoDBAttribute(attributeName = "password")
    var password: String? = null
        private set
    @get:DynamoDBAttribute(attributeName = "category")
    var category: String? = null
        private set

    fun phone(_phone: String) {
        this.phone = _phone
    }

    fun email(_email: String) {
        this.email = _email
    }

    fun age(_age: String) {
        this.age = _age
    }

    fun city(_city: String) {
        this.city = _city
    }

    fun name(_name: String) {
        this.name = _name
    }

    fun password(_password: String) {
        this.password = _password
    }

    fun category(_category: String) {
        this.category = _category
    }

}
